docs/design-docs/mcp/observe/screen-streaming.md

```
@@ -91,11 +91,24 @@ New Unix socket: `~/.auto-mobile/video-stream.sock`
 
 ### Connection Handshake
 
+One JSON line each way; everything after the acknowledgement is binary framing.
+
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)